toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -60,9 +60,9 @@ gzip-1.9-5.cm1.x86_64.rpm
 make-4.2.1-5.cm1.x86_64.rpm
 mariner-release-1.0-20.cm1.noarch.rpm
 patch-2.7.6-7.cm1.x86_64.rpm
-util-linux-2.32.1-4.cm1.x86_64.rpm
-util-linux-devel-2.32.1-4.cm1.x86_64.rpm
-util-linux-libs-2.32.1-4.cm1.x86_64.rpm
+util-linux-2.32.1-3.cm1.x86_64.rpm
```

Comment:
Not sure what's happening here, but I think you can safely revert all changes to these .txt files, since you haven't modified any of these packages.

Reply:
Yea git was complaining about a merge conflict, so I thought this would fix it. Guess I should just revert the changes here, since now it's also complaining about the cgmanifests.